src/Park.js

```diff
@@ -24,22 +24,49 @@ class Park extends Component {
 
   render() {
     let imagePath = `./${this.props.selectedPark.image}`;
-    return (
-      <div className="card-overlay">
-        <div className="park-card-small"> 
-          <i className="far fa-times-circle" onClick={this.removeCard}></i>
-          <h1 className="park-title">{this.props.selectedPark.parkName} National Park</h1>
-          <img className="park-img" src={imagePath} />
-          {/* {imageSource} */}
-          <h3 className="park-text">{this.props.selectedPark.description.substring(0, 80)}...</h3>
-          <button className="button-small" onClick={this.showFullCard}>View More</button>
-          <div className="user-list-btns">
-          <Buttons iconType="fas fa-hiking" toolTipText="Add to Visited Parks" toolTipSide="left" parkUrl={this.props.selectedPark.urlCode} storageKey="visitedParks"/>
-          <Buttons iconType="fas fa-clipboard-list" toolTipText="Add to Bucket List" toolTipSide="right" parkUrl={this.props.selectedPark.urlCode} storageKey="bucketList"/>
+    switch(this.state.displayFull) {
```

Comment:
The full view of the park looks awesome! Thank you for setting this up and styling!